libs/libcommon/cmake/find_jemalloc.cmake

```diff
@@ -1,13 +1,19 @@
-option (ENABLE_JEMALLOC "Set to TRUE to use jemalloc" ON)
-option (USE_INTERNAL_JEMALLOC_LIBRARY "Set to FALSE to use system jemalloc library instead of bundled" ${NOT_UNBUNDLED})
+# bundled jemalloc only for linux x64
+if (NOT ARCH_FREEBSD AND NOT APPLE)
+    option (ENABLE_JEMALLOC "Set to TRUE to use jemalloc" ON)
```

Comment:
Why we can't enable jemalloc on Mac?

Reply:
We can, unbundled from brew.
or run configure in compile-time for generating proper
contrib/jemalloc-cmake/include_linux_x86_64/jemalloc/internal/jemalloc_internal_defs.h